cluster-stamp.json

```diff
@@ -26,6 +26,13 @@
                 "description": "The certificate data for app gateway TLS termination. It is base64"
             }
         },
+        "clusterAuthorizedIPRanges": {
+            "type": "array",
+            "defaultValue": [],
```

Comment:
if we deploy using this default value what is the result? Nobody can contact the `Kubernetes API Server` or anyone?

Reply:
I've just seen it is `anyone`. Resolving this conversation